### ISSUE TITLE
setCampaignFromUrl() not working

### ISSUE THREAD

**Issue by okanari2** [appfeel/analytics-google#7]
Plugin works very well, pushes data to GA, but I neet to set campaing tracking variables (like utm_source etc) and I am using setCampaignFromUrl() methot to send campaign URL to GA. Nothing changes, there is no campaign data in GA(Google Anayitcs)

Could you please check the issue?

**Comment by philecom** [comment]
Yeah, I am also facing the same issue. @okanari2  Where you able to resolve it?

**Comment by miqmago** [comment]
Sorry for late response, have had an unmantained time...

Is your campaign url like this one?

``` java
String campaignData = "http://examplepetstore.com/index.html?" +
    "utm_source=email&utm_medium=email_marketing&utm_campaign=summer" +
    "&utm_content=email_variation_1";
```

Normally it should work, but if there is a builder which does not implement `setCampaignParamsFromUrl` method, then it will fail. Which method are you trying to track with campaign url? (is it an event, an screen view, etc...)
